BrowserKit/Sources/ToolbarKit/AddressToolbar/AddressToolbarUXConfiguration.swift

```diff
@@ -12,44 +12,52 @@ public struct AddressToolbarUXConfiguration {
     let locationTextFieldTrailingPadding: CGFloat
     let shouldBlur: Bool
     let backgroundAlpha: CGFloat
+    let alpha: CGFloat
```

Comment:
Can we change the name of this `alpha` so that it's more descriptive. At the moment it's hard to understand what is the difference between `alpha` and `backgroundAlpha` (a line above) without looking into the code further. It should probably be renamed everywhere in this PR.